rskj-core/src/test/java/co/rsk/peg/BridgeSupportTestIntegration.java

```diff
@@ -3148,330 +3144,31 @@ void getRetiringFederationWallet_nonEmpty() throws IOException {
-                "04641fb250d7ca7a1cb4f530588e978013038ec4294d084d248869dd54d98873e45c61d00ceeaeeb9e35eab19fa5fbd8f07cb8a5f0ddba26b4d4b18349c09199ad"
-        )).getAddress();
-        RskAddress sender = new RskAddress(senderBytes);
-        when(mockedTx.getSender(any(SignatureCache.class))).thenReturn(sender);
-        LockWhitelist mockedWhitelist = mock(LockWhitelist.class);
-        when(mockedWhitelist.isDisableBlockSet()).thenReturn(true);
-        BridgeSupport bridgeSupport = getBridgeSupportWithMocksForWhitelistTests(mockedWhitelist);
-
-        BigInteger disableBlockDelayBI = BigInteger.valueOf(100);
-        Assertions.assertEquals(-1, bridgeSupport.setLockWhitelistDisableBlockDelay(mockedTx, disableBlockDelayBI).intValue());
-        verify(mockedWhitelist, never()).put(any(), any());
-    }
-
-    @Test
-    void setLockWhitelistDisableBlockDelay_notAuthorized() throws IOException, BlockStoreException {
-        Transaction mockedTx = mock(Transaction.class);
-        byte[] senderBytes = Hex.decode("0000000000000000000000000000000000aabbcc");
-        RskAddress sender = new RskAddress(senderBytes);
-        when(mockedTx.getSender(any(SignatureCache.class))).thenReturn(sender);
-        LockWhitelist mockedWhitelist = mock(LockWhitelist.class);
-        BridgeSupport bridgeSupport = getBridgeSupportWithMocksForWhitelistTests(mockedWhitelist);
-
-        BigInteger disableBlockDelayBI = BigInteger.valueOf(100);
-        Assertions.assertEquals(-10, bridgeSupport.setLockWhitelistDisableBlockDelay(mockedTx, disableBlockDelayBI).intValue());
-        verify(mockedWhitelist, never()).put(any(), any());
-    }
-
-    @Test
-    void removeLockWhitelistAddress_ok() {
-        Transaction mockedTx = mock(Transaction.class);
-        byte[] senderBytes = ECKey.fromPublicOnly(Hex.decode(
-                // Public key hex of the authorized whitelist admin in regtest, taken from BridgeRegTestConstants
-                "04641fb250d7ca7a1cb4f530588e978013038ec4294d084d248869dd54d98873e45c61d00ceeaeeb9e35eab19fa5fbd8f07cb8a5f0ddba26b4d4b18349c09199ad"
-        )).getAddress();
-        RskAddress sender = new RskAddress(senderBytes);
-        when(mockedTx.getSender(any(SignatureCache.class))).thenReturn(sender);
-        LockWhitelist mockedWhitelist = mock(LockWhitelist.class);
-        BridgeSupport bridgeSupport = getBridgeSupportWithMocksForWhitelistTests(mockedWhitelist);
-
-        when(mockedWhitelist.remove(any(Address.class))).then((InvocationOnMock m) -> {
-            Address address = m.<Address>getArgument(0);
-            Assertions.assertEquals("mwKcYS3H8FUgrPtyGMv3xWvf4jgeZUkCYN", address.toBase58());
-            return true;
-        });
-
-        Assertions.assertEquals(1, bridgeSupport.removeLockWhitelistAddress(mockedTx, "mwKcYS3H8FUgrPtyGMv3xWvf4jgeZUkCYN").intValue());
-    }
+        List<LockWhitelistEntry> entries = Arrays.stream(new Integer[]{2, 3, 4, 5})
```

Comment:
this can be made more concise as:

```suggestion
        List<LockWhitelistEntry> entries = IntStream.rangeClosed(2, 5)
```

Reply:
Thanks so much, @apancorb! This test case was removed.